packages/web-components/src/components/cbp-toast/cbp-toast.tsx

```diff
@@ -0,0 +1,63 @@
+import { Component, Prop, Host, Watch, h } from '@stencil/core';
+
+@Component({
+  tag: 'cbp-toast',
+  styleUrl: 'cbp-toast.scss'
+})
+export class CbpToast {
+
+  /** specifies the icon loaded into the sidebar */
+  @Prop() icon: string = 'user';
+
+  /** specifies the color for the toast */
+  @Prop({ reflect: true }) color: 'info' | 'danger' | 'success' | 'warning' = 'info';
+
+  /** specifies time in seconds for the toast to be displayed */
+  @Prop() timer: 3 | 5 | 10;
+
+  /** When set, specifies that the toast is open */
+  @Prop({ reflect: true }) open: boolean = true;
```

Comment:
booleans default to false and you should never default them to true because in plain HTML, the absence of a boolean attribute is false. So how do you set it to false? Maybe it makes sense in this case if the component is handling the dismissal but we should talk it through.

Reply:
updated to remove the default setting on the toast.tsx and add the storybook control to start the sample with this populated as true, as discussed on call